docs/units/quantity.rst

```diff
@@ -447,26 +500,25 @@ It can also be used for in-place conversion::
 Subclassing Quantity
 ====================
 
-To subclass |quantity|, one generally proceeds as one would when subclassing
-:class:`~numpy.ndarray`, i.e., one typically needs to override ``__new__``
-(rather than ``__init__``) and uses the ``numpy.ndarray.__array_finalize__``
-method to update attributes.  For details, see the `numpy documentation on
-subclassing
-<https://docs.scipy.org/doc/numpy/user/basics.subclassing.html>`__.  For
-examples, one can look at |quantity| itself, where, e.g., the
+To subclass |Quantity|, you generally proceed as you would when subclassing
+:class:`~numpy.ndarray` (i.e., you typically need to override ``__new__``,
+rather than ``__init__``, and use the ``numpy.ndarray.__array_finalize__``
+method to update attributes). For details, see the `NumPy documentation on
+subclassing <https://docs.scipy.org/doc/numpy/user/basics.subclassing.html>`__.
+For examples, you can look at |Quantity| itself, where, for example, the
```

Comment:
Two "for example" in the same sentence.  Maybe "To get a sense of what is involved, have a look ..."?